src/framework/shortcuts/view/editshortcutmodel.cpp

```diff
@@ -87,7 +87,8 @@ void EditShortcutModel::inputKey(Qt::Key key, Qt::KeyboardModifiers modifiers)
     }
 
     // remove shift-modifier for keys that don't need it: letters and special keys
```

Comment:
Shouldn't that commene rather be `// remove shift-modifier for keys that don't need it: all non-letters, except a few special keys'